src/Components/WebAssembly/Samples/HostedBlazorWebassemblyApp/Client/wwwroot/HostedBlazorWebassemblyApp.Client.lib.module.js

```diff
@@ -0,0 +1,7 @@
+export function beforeStart(serverOptions) {
+    console.log('starting!');
+}
+
+export function afterStarted(blazor) {
+    console.log('started!');
+}
```

Comment:
Could this be removed from the sample? Should be enough to have it in the E2E test app.